components/stats/WordLearningStatusBarChart.tsx

```diff
@@ -25,7 +26,7 @@ const WordLearningStatusBarChart = () => {
     fulfilledTimeStamp,
   } = useGetWordLearningStatsQuery(undefined, { pollingInterval: STAT_POLLING_INTERVAL });
 
-  if (isLoading) return null;
+  if (isLoading) return <CardSkeleton count={1} height={height} width={width} />;
```

Comment:
How would it be if this was isFetching like we did in other components?

Reply:
I knowingly did it that way, because I don't want the loading screen to show up each time